app/src/main/java/com/example/marvelapp/mvvm/model/CharacterListModel.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.marvelapp.mvvm.model
+
+import com.example.marvelapp.R
+
+class CharacterListModel {
+    private var characterList = mutableListOf<Character>()
+
+    fun sendCharacterList(): MutableList<Character> {
```

Comment:
please change the method's name. I think that should be more appropriated `getCharacterList`